### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by bashleigh**
#1 #7

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - "8"
+  - "9"
+services:
+  - docker
+before_install:
+  - npm i -g npm@latest
+  - npm i -g yarn
+install:
+  - yarn
+script:
```
Comment (fenos) [inline .travis.yml]: probably you'll need to lunch docker-compose before running the tests? 
Or at least install rabbitmq in the vm

File: src/__tests__/amqp.module.spec.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      ],
+    }).compile();
+
+    const connection = module.get<any>('amqpConnection_default');
+
+    expect(connection).toBeTruthy();
+    connection.close();
+  });
+
+  it('Load module using env', async () => {
```
Comment (fenos) [inline src/__tests__/amqp.module.spec.ts]: this test is not actually testing the functionality with the `.env` file? i can't see that file.

File: src/__tests__/amqp.module.spec.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            name: 'test',
+          },
+        ]),
+      ],
+    }).compile();
+
+    const connection1 = module.get<any>('amqpConnection_0');
+    const connectionTest = module.get<any>('amqpConnection_test');
+
+    expect(connection1).toBeTruthy();
```
Comment (fenos) [inline src/__tests__/amqp.module.spec.ts]: probably better asserting `instanceOf`

File: src/__tests__/amqp.module.spec.ts (hunk 2 above)
Comment (bashleigh) [reply src/__tests__/amqp.module.spec.ts]: At the moment there isn't a config file for this, the default is `localhost:5672` from connection

File: .travis.yml (hunk 1 above)
Comment (bashleigh) [reply .travis.yml]: There is a docker-compose file which just needs `docker-compose up -d` to be run before running tests. I believe travis accepts this config for docker containers

```yml
services:
  - docker
```

**Pull request merged**